helix-core/src/main/java/org/apache/helix/cloud/event/helix/HelixEventHandlingUtil.java

```diff
@@ -0,0 +1,56 @@
+package org.apache.helix.cloud.event.helix;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.helix.BaseDataAccessor;
+
+
+ class HelixEventHandlingUtil {
+
+  /**
+   * Enable or disable an instance for cloud event.
+   * It will enable/disable Helix for that instance. Also add the instance cloud event info to
+   * clusterConfig Znode when enable.
+   * @param clusterName
+   * @param instanceName
+   * @param message
+   * @param isEnable
```

Comment:
nit: enabled or isEnabled would be better

Reply:
This function will enable or disable an instance. would `isEnable` be better to indicate the operation type? I feel like "enabled" is used to describe the state if an instance, not the operation type.

Reply:
I'm fine either way. Feel free to resolve this.